### ISSUE TITLE
Provide feedback on the scoring method

### ISSUE THREAD

**Issue by kylefox1** [Ledgerback/DGSF#20]
The current scoring method for each response is:

- Yes: 100
- No: 0
- N/A: Redistribute points
- Does not answer question: 0
- Partial: 50

**Comment by linxule** [comment]
> The points awarded for each response is:
> 
> 
> 
> (response: points)
> 
> 
> 
> - Yes: 100
> 
> - No: 0
> 
> - N/A: Redistribute points
> 
> - Does not answer question: 0
> 
> - Partial: 50
> 
> 
> 
> The total score is calculated by summing the points for all responses
> 
> 
> 
> I will provide a notebook on Google Colab to test out some scoring methods.

What does the "redistribute points" here mean? Like the points will be added to other questions or?

**Comment by kylefox1** [comment]
> What does the "redistribute points" here mean? Like the points will be added to other questions or?

Yes. The points are redistributed to the applicable questions under the principle.
